hw/net/rocker/rocker_hw.h

```diff
@@ -475,6 +504,16 @@ enum rocker_of_dpa_overlay_type {
     (ROCKER_GROUP_TYPE_SET(ROCKER_OF_DPA_GROUP_TYPE_L3_UCAST) |\
      ROCKER_GROUP_INDEX_LONG_SET(index))
 
+/* cmd info (nested tlvs) for P4 */
+enum {
+    ROCKER_TLV_P4_RMT_INFO_UNSPEC = 0,  /* don't use 0 for tlv type */
```

Comment:
I wanted to have something in TLV that tells me that this is a nested TLV
(so it won't be used incorrectly).
I think, I wanted to drop INFO and keep CMD but did the reverse by mistake.
E.g. I wanted -
ROCKER_TLV_P4_RMT_CMD_UNSPEC, ..
This will be consistent with -
ROCKER_TLV_CMD_PORT_SETTINGS_UNSPEC

Eitherways.. it really does not shorten the name. I can drop both INFO and
CMD (altough prefer to keep some indication that this is nested tlv under
CMD)

On Sun, Apr 19, 2015 at 9:12 PM, Scott Feldman <notifications@github.com>
wrote:

> In hw/net/rocker/rocker_hw.h
> <https://github.com/pbhide/qemu-rocker/pull/1#discussion_r28663229>:
>
> > @@ -475,6 +504,16 @@ enum rocker_of_dpa_overlay_type {
> >      (ROCKER_GROUP_TYPE_SET(ROCKER_OF_DPA_GROUP_TYPE_L3_UCAST) |\
> >       ROCKER_GROUP_INDEX_LONG_SET(index))
> >
> > +/* cmd info (nested tlvs) for P4 */
> > +enum {
> > +    ROCKER_TLV_P4_RMT_INFO_UNSPEC = 0,  /* don't use 0 for tlv type */
>
> Can probably drop the "INFO" part of the name to keep things shorter and
> be consistent with OF-DPA cmd info tlvs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbhide/qemu-rocker/pull/1/files#r28663229>.
>